ingestion/src/metadata/profiler/interface/sqlalchemy/profiler_interface.py

```diff
@@ -305,6 +286,9 @@ def _(
         Returns:
             dictionnary of results
         """
+        session = kwargs.get("session")
+        column = kwargs.get("column")
```

Comment:
if we need both the session and column can we add them as explicit parameters?

Reply:
@pmbrull we can. I was trying to comply with the parent method signature -- as mypy was complaining. Let me know what you think

Reply:
then we could add them to the parent as well right? I imagine that any metric computation would always need the `session` and `columns` so we could define it as such in the abs

Reply:
Not necessarily. data lake does not need a session object and `column` is table-specific.